README.md

```diff
@@ -100,8 +100,7 @@ Ready for use it.
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Finish the mobile version]**
-- [ ] **[Desktop Version]**
+- [ ] **[Add Functionality]**
```

Comment:
- [optional] please kindly remove the live demo section for now, since you are yet to deploy your project. This can be added when you do so